packages/storybook8/stories/Components/UnsupportedBrowser/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { UnsupportedBrowser as UnsupportedBrowserComponent } from '@azure/communication-react';
+import { Meta } from '@storybook/react';
+import { hiddenControl } from '../../controlsUtils';
+import { UnsupportedBrowserExamples } from './snippets/UnsupportedBrowserExamples.snippet';
+import { UnsupportedEnvironmentModals } from './snippets/UnsupportedEnvironmentModal.snippet';
+
+export const UnsupportedBrowserExamplesDocsOnly = {
+  render: UnsupportedBrowserExamples
+};
+
+export const UnsupportedEnvironmentModalsDocsOnly = {
+  render: UnsupportedEnvironmentModals
+};
+
+export { UnsupportedBrowser } from './UnsupportedBrowser.story';
+
+const meta: Meta = {
+  title: 'Components/Unsupported Browser',
+  component: UnsupportedBrowserComponent,
+  argTypes: {
+    // hidden controls
+    onTroubleshootingClick: hiddenControl,
+    strings: hiddenControl
+  }
```

Comment:
Maybe more of a question @prabhjot-msft or @JamesBurnside but if we are hiding all the controls, there is nothing for readers to really interact with. The story looks incomplete
![image](https://github.com/user-attachments/assets/23e76cf0-a62d-43f7-86c5-f9d04d1a58ac)

Reply:
I don’t think there’s any control you can interact with for this component and It was also the case for old storybook page. But I might be wrong though since this is a component for calling

Reply:
We might be able to refactor the page so these components are part of the story. we do want contoso to be able to see these components in a place that isn't a picture so if we put them in a `Canvas` that could be helpful

Reply:
Yeah if there are no controls we should move it to canvas inside the docs

Reply:
since the doc is currently showing the component, so i removed the story: 
<img width="353" alt="image" src="https://github.com/user-attachments/assets/8fc4c93c-fb22-46e0-acb5-c5d25c475e31">


please take a look again, thanks! @dmceachernmsft @prabhjot-msft